build/karma.conf.js

```diff
@@ -2,7 +2,7 @@
 const webpackConfig = require('./webpack.karma.config');
 
 const DOC_STATIC_ASSETS_VERSION = '0.130.0';
-const MEDIA_STATIC_ASSETS_VERSION = '0.127.0';
+const MEDIA_STATIC_ASSETS_VERSION = '1.5.1';
```

Comment:
When you do manage to merge this in after Shaka 2.2, please make sure to update this to the latest version at that point. Right now, it's 1.6.2

Reply:
Will do